### PR TITLE
chore(#8181): remove old action bar

### PR DESCRIPTION
**Pull request by latin-panda**
# Description

- Removes action bar component
- Removes old permissions 
- Cleans CSS, HTML, JS, Specs files
- Removed action bar e2e tests to make build pass, but I didn't add any new ones. The automation tests work will be done in https://github.com/medic/care-teams/issues/148 by @kapelo 

#8181
https://github.com/medic/care-teams/issues/126

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs
If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8181-remove-old-actionbar/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8181-remove-old-actionbar/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8181-remove-old-actionbar/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by latin-panda** [comment]
@Benmuiruri @kapelo The heavy work is done, but I still need to test it more and clean it. 
I would appreciate your feedback at this stage, especially if there's unused code to remove (variables, functions, imports, etc).

File: tests/e2e/default/contacts/fab-actions.wdio-spec.js
Comment (latin-panda) [inline tests/e2e/default/contacts/fab-actions.wdio-spec.js:1]: This was just a file renaming, and the action bar section was removed.
The test cases you see here are the original ones.

**Comment by latin-panda** [comment]
@Benmuiruri @kapelo I finished cleaning the PR. It's ready for a review

File: webapp/src/ts/modules/reports/reports-more-menu.component.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<section class="more-options-menu-container" *ngIf="(displayExportOption() || displayDeleteOption() || displayEditOption()) && !useOldActionBar">
+<section class="more-options-menu-container" *ngIf="(displayExportOption() || displayDeleteOption() || displayEditOption())">
```
Comment (Benmuiruri) [inline webapp/src/ts/modules/reports/reports-more-menu.component.html:1]:
```suggestion
<section
  class="more-options-menu-container"
  *ngIf="(displayExportOption() || displayDeleteOption() || displayEditOption())">
```

**Review by Benmuiruri — CHANGES_REQUESTED** [review]
Looking good, great effort tracking down all those instances. 

I found 1 ghost 👻  function.

File: webapp/src/ts/modules/contacts/contacts-more-menu.component.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<section class="more-options-menu-container" *ngIf="(displayExportOption() || displayDeleteOption() || displayEditOption()) && !useOldActionBar">
+<section class="more-options-menu-container" *ngIf="(displayExportOption() || displayDeleteOption() || displayEditOption())">
```
[the remark this answers is not in the document]
Comment (latin-panda) [reply webapp/src/ts/modules/contacts/contacts-more-menu.component.html]: Resolved!

File: webapp/src/ts/modules/reports/reports-more-menu.component.html (hunk 1 above)
Comment (latin-panda) [reply webapp/src/ts/modules/reports/reports-more-menu.component.html]: Resolved!

**Review by Benmuiruri — APPROVED** [review]
LGTM!

**Review by kapelo — APPROVED** [review]
LGTM

**Pull request merged**